src/ssa/dynobj_instance_analysis.cpp

```diff
@@ -223,7 +223,14 @@ void dynobj_instance_domaint::transform(
         remove_dereferences(assignment.lhs(), instances);
         add_aliased_dereferences(assignment.lhs(), instances);
 
-        live_pointers[v.symbol_expr()].insert(rhs);
+        // Do not include CPROVER objects
+        // TODO: do it better than check for "malloc" substring
+        if(!(rhs.id()==ID_symbol &&
+             id2string(to_symbol_expr(rhs).get_identifier()).find(
+               "malloc")!=std::string::npos))
```

Comment:
That's very brittle. Can you make sure this doesn't clash with user variables please?